workloads/sample/sample_run.py

```diff
@@ -93,6 +93,9 @@ def get_total_os_memory():
 workload.options.create_trigger = db_size_target_gb * 1024
 workload.options.create_target = db_size_target_gb * 1024
 
+# Limit the number of files.
```

Comment:
NP: maybe add "that can be open" to be more clear it is not an overall generated amount?

Reply:
Sure, see 5507f14d014c69d39559cc59ef9263f29cbe3b13